src/EA.Weee.RequestHandlers.Tests.Unit/Scheme/MemberRegistration/ProcessXMLFileHandlerTests.cs

```diff
@@ -194,7 +199,7 @@ public async void ProcessXmlfile_BusinessErrors_TriesToCalculateCharges()
             
             await handler.HandleAsync(Message);
 
-            A.CallTo(() => totalChargeCalculator.TotalCalculatedCharges(Message, scheme, A<int>.Ignored, A<bool>._, ref totalCharges)).MustHaveHappened(Repeated.Exactly.Once);
+            A.CallTo(() => totalChargeCalculator.TotalCalculatedCharges(Message, A<Scheme>.Ignored, A<int>.Ignored, A<bool>._, ref totalCharges)).MustHaveHappened(Repeated.Exactly.Once);
```

Comment:
Probably worth updating this test to ensure that TotalCalculatedCharges is called with the correct values (i.e. scheme, year etc) rather than not indicating they can be ignored